frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/vm-settings-tab-validation.ts

```diff
@@ -95,6 +112,48 @@ export const validateOperatingSystem: VmSettingsValidator = (field) => {
   return asValidationObject(`Select matching for: ${guestFullName}`, ValidationErrorType.Info);
 };
 
+const memoryValidation: VmSettingsValidator = (field, options) => {
+  const memValueGB = iGetFieldValue(field);
+  if (!memValueGB) {
+    return null;
+  }
+  const memValueBytes = memValueGB * 1024 ** 3;
+  const validations = getTemplateValidations(options, getFieldId(VMSettingsField.MEMORY));
+  if (validations.length === 0) {
+    return null;
+  }
+
+  let errorMsg;
+
+  /*
+  - Check if at least one validation has passed.
+  - Would *NOT* check if all the validations have passed since it may not be possible
+    For example:
+    If we have two 'between' validations: 2-4 and 6-10 from 2 different templates,
+    There is no value that would satisfy both validations.
+  */
+  const isValid = validations.some((vArr) => {
+    const ivo: IntegerValidationObj = getIntegerValidationType(vArr);
+    errorMsg = ivo.message;
+    switch (ivo.type) {
+      case IntegerValidtionType.between:
+        return validateBetweenRule(ivo, memValueBytes);
```

Comment:
sinse we know that the current rule vArr ( should be one not an array ) is a "between" rule, then vArr must have a min and max properties, we should be able to write: `memValueBytes >= vArr.min && vArr.max <= memValueBytes`, what am I missing ?

Reply:
commented above. it should be an array.